data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/Main.java

```diff
@@ -52,6 +55,13 @@
    */
   public static void main(final String[] args) {
 
+    // HACK HACK HACK
+    // maven-shade-plugin doesn't include the LogstashEncoder class, neither by specifying the
+    // dependency with scope `provided` nor `runtime`, and adding include rules to
+    // maven-shade-plugin.
+    // Instantiating an Encoder here we force it to include the class.
```

Comment:
What about using include directives? http://maven.apache.org/plugins/maven-shade-plugin/examples/includes-excludes.html
So you force to include that class

Reply:
I tried never got it working :cry:

Reply:
Ok let's keep it this way and we'll figure out later